### DISCUSSION TITLE
Subtasks for a measurable habits

### DISCUSSION THREAD

**Discussion by dharux** [iSoron/uhabits#1979] [Feature Requests]
Love the app, I use it everyday and it really helps me. 

A feature that I would really like is to divide a measurable habit into multiple sub-tasks. Then, one would set a target for the main habit (or super task) and all the measurements from the subtasks would contribute towards the main target. This would make it easier to group similar tasks, and not have to set individual targets for each but a combined one. 

An example could be: 
Habit: Study physics
- Study relativity 
- Study quantum mechanics 
- Study thermodynamics 
- ... 

The target could be a number of hours spent on the main task and all subtasks contribute to it. 
One could also use this for projects which have multiple parallel aspects or for practicing multiple instruments. 
An approach like this would allow for flexible and dynamic time allotment to each of the subtasks rather than a fixed target per week or month.

**Comment by ezio-melotti** [comment]
See #495.
